float64[32] sensorValues
int64 treeIndex
---
float64[2] actuatorValues
